cgi-bin/hello.py

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env python3
+
+print("Content-type: text/html")
+print()
+print("<h1>Hello Python!</h1>")
```

Comment:
Please add blank line to the end of the file

Reply:
Added